Workers/GameLogic/ReceptionistOptions.cs

```diff
@@ -5,14 +5,14 @@
 namespace GameLogic
 {
     [Verb("receptionist", HelpText = "Connect to a deployment using the receptionist")]
-    internal class ReceptionistOptions : IWorkerOptions, IReceptionistOptions
+    internal class ReceptionistOptions : IReceptionistOptions
     {
-        public string WorkerName { get; set; }
-        public string LogFileName { get; set; }
+        public string? WorkerName { get; set; }
+        public string? LogFileName { get; set; }
+
         [Value(0, Hidden = true)] // Stops "pos value 0" being shown when help is ran
-        public IEnumerable<string> UnknownPositionalArguments { get; set; }
-        public string SpatialOsHost { get; set; }
+        public IEnumerable<string> UnknownPositionalArguments { get; set; } = null!;
+        public string SpatialOsHost { get; set; } = null!;
```

Comment:
Default to `localhost`?

Reply:
All the defaults are handled in `IReceptionistOptions` via the `[Option(Default='')]` attribute
This just tells the compiler not to complain that they're not initialized